### DISCUSSION TITLE
Installation Executable CLI Parameters

### DISCUSSION THREAD

**Discussion by AdreKiseque** [git-for-windows/git#5335] [Q&A]
I understand the installation executable accepts command line arguments for configuring the installation, but I haven't been able to find information on them anywhere in the documentation. Does anyone know where to look?

**Comment by rimrul** [comment]
https://jrsoftware.org/ishelp/index.php?topic=setupcmdline

https://github.com/git-for-windows/git/wiki/Silent-or-Unattended-Installation

https://github.com/git-for-windows/git/wiki/Mapping-between-Git-Installer-GUI-Settings-And-Command-Line-Arguments

**Reply by AdreKiseque** [reply]
Ah, I hadn't seen there was a wiki! Thank you!
